sources/Core/Numerics/Matrix3x3.cs

```diff
@@ -129,7 +129,8 @@ public static Matrix3x3 Transpose(Matrix3x3 m)
             return new Matrix3x3(
                 new Vector3(m.X.X, m.Y.X, m.Z.X),
                 new Vector3(m.X.Y, m.Y.Y, m.Z.Y),
-                new Vector3(m.X.Z, m.Y.Z, m.Z.Z));
+                new Vector3(m.X.Z, m.Y.Z, m.Z.Z)
+              );
```

Comment:
These all seem to be getting two extra spaces, they should be aligned with the `return`:
```suggestion
            );
```

Reply:
fixed